### PR TITLE
Added Processing in P

### PR DESCRIPTION
**Pull request by BabbarRaghav**

### CODE REVIEW

File: A-Z-checkboxes.html [hunk 1]
```diff
@@ -1453,6 +1453,9 @@ <h1>P</h1>
 <input type="checkbox" id="PL/C" name="pchbx" value="PL/C" />
 <label for="PL/C"> PL/C </label><br />
 
+<input type="checkbox" id="Processing" name="pchbx" value="Processing" />
+<label for="PL/C"> Processing </label><br />
```
Comment (chilarai) [inline A-Z-checkboxes.html:1457]: Change the `label for="PL/C"`

**Review by chilarai — CHANGES_REQUESTED** [review]
Please make the required changes

**Review by BabbarRaghav — COMMENTED** [review]
done

**Review by chilarai — APPROVED** [review]
Thank you

**Pull request merged**